RangeRequest.bs

```diff
@@ -10,26 +10,218 @@ Editor: Chris Lilley, W3C, https://svgees.us/, w3cid 1438
 Editor: Myles C. Maxfield, Apple Inc., mmaxfield@apple.com, w3cid 77180
 Editor: Garret Rieger, Google Inc., grieger@google.com
 Abstract: Specification for incremental font transfer via HTTP range requests.
-Status Text: This is a largely empty document because we have just started working on it.
+Status Text: This is an editor's draft.
 </pre>
 
+<h2 id="intro">
+Introduction</h2>
 
-Introduction {#intro}
-=====================
+<em>This section is not normative.</em>
+
+Incremental Font Transfer (IFT) is a collection of technologies to improve the latency of remote fonts (or "web fonts") on the web. Without this technology, a browser needs to download every last byte of a font before it can render any characters using that font. IFT allows the browser to download only some of the bytes in the file, thereby decreasing the perceived latency between the time when a browser realizes it needs a font and when the necessary text can be rendered with that font.
+
+There are two technologies inside the IFT collection of technologies. This document describes one of them: the "Range-request method." In this method, the browser uses HTTP range requests to download only specific regions of the font file that it needs. In order for this method to be as effective as possible, the font file itself should be arranged in a particular way, in order to decrease the number of requests the browser needs to make.
+
+The range-request IFT method was specifically designed to target good performance on Chinese, Japanese, and Korean languages. These languages have the largest font files, and include many glyphs which are generally independent. Thus, these languages are a good target for selective downloads.
```

Comment:
This doesn't seem correct, because for slower networks the performance is worse.

Reply:
Also, the claim that CJK fonts "include many glyphs which are generally independent" may not hold for fonts that have large number of composite glyphs.